pkgs/tools/misc/infracost/default.nix

```diff
@@ -10,15 +10,56 @@ buildGoModule rec {
     repo = "infracost";
     sha256 = "sha256-3AH/VUKIno/jObep5GNfIpyOW5TcfZ5UZyornJWTGOw=";
   };
-
   vendorSha256 = "sha256-zMEtVPyzwW4SrbpydDFDqgHEC0/khkrSxlEnQ5I0he8=";
 
+  ldflags = [ "-s" "-w" "-X github.com/infracost/infracost/internal/version.Version=v${version}" ];
+
+  # Install completions post-install
+  nativeBuildInputs = [ installShellFiles ];
+
   checkInputs = [ terraform ];
-  checkPhase = "make test";
+  checkPhase = ''
+    runHook preCheck
+    make test
+    runHook postCheck
+  '';
+
+  postInstall = ''
+    # panic if .config directory can't be accessed
+    # https://github.com/infracost/infracost/pull/862
+    export HOME="$TMPDIR"
+    mkdir -p "$HOME/.config/infracost"
+    export INFRACOST_SKIP_UPDATE_CHECK=true
+
+    installShellCompletion --cmd infracost \
+      --bash <($out/bin/infracost completion --shell bash) \
+      --fish <($out/bin/infracost completion --shell fish) \
+      --zsh <($out/bin/infracost completion --shell zsh)
+  '';
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+
+    export HOME="$TMPDIR"
+    mkdir -p "$HOME/.config/infracost"
+    export INFRACOST_SKIP_UPDATE_CHECK=true
+
+    $out/bin/infracost --help
+    $out/bin/infracost --version | grep "v${version}"
+
+    runHook postInstallCheck
+  '';
 
   meta = with lib; {
+    homepage = "https://infracost.io";
+    changelog = "https://github.com/infracost/infracost/releases/tag/v${version}";
     description = "Cloud cost estimates for Terraform in your CLI and pull requests";
-    homepage = "https://github.com/infracost/infracost";
+    longDescription = ''
+      Infracost shows hourly and monthly cost estimates for a Terraform project.
+      This helps developers, DevOps et al. quickly see the cost breakdown and
+      compare different deployment options upfront.
+    '';
     license = [ licenses.asl20 ];
     maintainers = [ maintainers.davegallant ];
```

Comment:
Are you going to add yourself as a maintainer?

Reply:
Ahh i thought i commit amend and forcepushed..
Ill fix this when i get back

Reply:
Fixed